evalml/tests/component_tests/test_delayed_features_transformer.py

```diff
@@ -84,12 +88,11 @@ def test_delayed_feature_extractor_maxdelay3_forecasthorizon1_gap0(
             "target_delay_4": y_answer.shift(4),
         }
     )
-
     assert_frame_equal(
         answer,
-        DelayedFeatureTransformer(max_delay=3, gap=0, forecast_horizon=1).fit_transform(
-            X=X, y=y
-        ),
+        DelayedFeatureTransformer(
+            max_delay=3, gap=0, forecast_horizon=1, conf_level=1.0
```

Comment:
Using a `conf_level` of 1.0 selects all lags. Some of these tests rely on the DelayedFeatureTransformer selecting all lags.